src/services/iasql.ts

```diff
@@ -775,20 +775,13 @@ ${Object.keys(tableCollisions)
   }
   // Sort the modules based on their dependencies, with both root-to-leaf order and vice-versa
   const rootToLeafOrder = sortModules(mods, existingModules);
-  const leafToRootOrder = [...rootToLeafOrder].reverse();
-  // Actually run the installation. First running all of the preinstall scripts from leaf-to-root,
-  // then all of the postinstall scripts from root-to-leaf. Wrapped in a transaction so any failure
-  // at this point when we're actually mutating the database doesn't leave things in a busted state.
+  // Actually run the installation. The install scripts are run from root-to-leaf. Wrapped in a
+  // transaction so any failure at this point when we're actually mutating the database doesn't leave things in a busted state.
   await queryRunner.startTransaction();
   try {
-    for (const md of leafToRootOrder) {
-      if (md.migrations?.preinstall) {
-        await md.migrations.preinstall(queryRunner);
-      }
-    }
     for (const md of rootToLeafOrder) {
-      if (md.migrations?.postinstall) {
-        await md.migrations.postinstall(queryRunner);
+      if (md.migrations?.install) {
+        await md.migrations.install(queryRunner);
       }
```

Comment:
Do we have a unit tests for this or a way of knowing if this will continue to work as expected with this change?

Reply:
So yes. We have the `dep-sorting` unit test and then every `module` test installs one-or-more modules during its own testing and would fail otherwise. Also the `simple-integration` test is designed to stress the add db, install modules, uninstall modules, remove db paths.